electron-app/main/window.ts

```diff
@@ -4,7 +4,8 @@ import MENU from './menu';
 import popupContextMenu from './contextMenu';
 import { APP_TITLE } from '../constants';
 
-const IS_DEV: boolean = process.env.NODE_ENV === 'development' || !process.env.NODE_ENV;
+const IS_DEV: boolean =
+  process.env.NODE_ENV === 'development' || !!process.env.IS_STAGING || !process.env.NODE_ENV;
```

Comment:
1. Staging is a separate environment and should not be confused with the `development` environment.
2. Prefer `process.env.NODE_ENV === 'staging'` for consistency